Header header
sensor_msgs/Image leftCamera
sensor_msgs/Image rightCamera
sensor_msgs/CameraInfo leftCameraInfo
sensor_msgs/CameraInfo rightCameraInfo
